src/sentry/runner/commands/run.py

```diff
@@ -354,3 +354,47 @@ def handler(signum, frame):
     signal.signal(signal.SIGINT, handler)
 
     subscriber.run()
+
+
+@run.command("pre-process-forwarder")
+@log_options()
+@click.option(
+    "--consumer-type",
+    default=None,
+    help="Specify which type of consumer to create, i.e. from which topic to consume messages.",
+    type=click.Choice(["events", "transactions", "attachments"]),
+)
+@click.option(
+    "--group",
+    default="pre-process-forwarder",
+    help="Kafka consumer group for the PreProcessForwarder. ",
+)
+@click.option(
```

Comment:
Can we make sure this supports the `max-batch-time-ms` parameter that Snuba supports? Without this value, it’s much harder to scale and run in lower volume environments like single tenant or on-premise without dropping this batch size value down to 1 in order to guarantee things get processed.

Reply:
Added parameter for configuring the maximum wait time for retrieving a batch of events.